packages/ui-toolkit/src/components/atoms/NumberInput/NumberInputStepper.tsx

```diff
@@ -27,7 +31,8 @@ const StepSuffixComponent = ({ step = 1, max = Number.POSITIVE_INFINITY, value,
 
       // @ts-ignore : to prevent onChange re writing as it can be passed by user
       //we are synthentically generating custome event to set value
-      onChange({ target: { value: floorValue } });
+      // sending actionType to detect that value has been increased by clicking on the + icon
+      onChange({ target: { value: floorValue }, actionType: 'INCREMENT' });
```

Comment:
INCREMENT should be taken out in a constant and keep it in the same folder as NumberInput.

Reply:
DONE